src/index.js

```diff
@@ -34,7 +34,7 @@ const createWorkerMiddleware = (worker) => {
       }
 
       return (action) => {
-        if (action.meta && action.meta.WebWorker) {
+        if (action.meta && action.meta.WebWorker === meta) {
```

Comment:
Let's make it more explicit:
```
if (
  action.meta &&
  (typeof workerName !== "undefined"
    ? action.meta.WebWorker === workerName
    : action.meta.WebWorker)
) {
  worker.postMessage(action);
}
```

Reply:
Ok, here might be a small problem - in this case middleware with no worker name would consume actions with explicit worker name. I'd like to be on the safe side in this case (so there's no unexpected side behaviors if someone uses both approaches in the same codebase).

Minor change:
```
? action.meta.WebWorker === workerName
    : action.meta.WebWorker === true)
```